cas-server-support-osf/src/main/java/io/cos/cas/web/flow/OpenScienceFrameworkLoginHandler.java

```diff
@@ -187,13 +209,24 @@ private boolean isInstitutionLogin(final RequestContext context) {
         return campaign != null && "institution".equals(campaign.toLowerCase());
     }
 
+    /**
+     * Obtain the institution ID in the request parameters for auto institution selection.
+     *
+     * @param context the request context
+     * @return the institution ID
+     */
+    private String getInstitutionIdFromRequestContext(final RequestContext context) {
```

Comment:
Renamed helper methods with suffix `FromRequestContext` in `OpenScienceFrameworkLoginHandler` to avoid confusion with the getters of `OpenScienceFrameworkLoginContext`.